### ISSUE TITLE
Vignette 1 with CAF-S1 and S4 numbers aren't matching

### ISSUE THREAD

**Issue by zkatmich** [soumelis-lab/ICELLNET#11]
Hello, I was running through the first vignette with the cytokine interaction in CAF-S1 and S4 and noticed my numbers in the figures aren't the same as shown in the vignette figures. It seems mostly to be a scaling issue but the p-value heat map was significantly different. Any feedback or assistance would be greatly appreciated. Thanks in advance.

**Comment by lmassenet-regad** [comment]
Hello, 
I would be happy to help but I need a bit of context/explanation of what you did and current issues to identify what is happening...

**Comment by zkatmich** [comment]
Yes, I downloaded the required files for the database, CC, and PC information from the Github and proceeded to copy the R code in the CAF vignette and run as I worked my way through it. I noticed in particular that the majority of the plots did not match the ones depicted in the vignette. I have included the versions I generated for reference. The main variation were the values in the balloon plot, several of the bar plots having widely different distributions, and the heatmap having quite different regions. Let me know if you need more information. Thanks for your help.
________________________________

**Comment by lmassenet-regad** [comment]
Hello, 
I cannot see what you generated, it is not attached to the previous comment, can you upload the results ?

I think I partially figured out what is happening. This tutorial has been created with an oldest version of the ligand/receptor database, so the plots are not up-to-date. I let you know when it is updated. This should affect the values of  global communication scores (as we take into account more interactions) and distributions, but the balloon plot values should not be affected.

**Comment by lmassenet-regad** [comment]
Thanks. 
I think there is a problem when running the gene.scaling function. In the tutorial: 
`PC.data=gene.scaling(data = PC.data, n=18, db = transform)`
and 
`CC.data= gene.scaling(data = data, n=4, db = db2)`

Normally, after applying this function, the matrix values should be between 0 and 10 for each gene. Maximum values of each  interaction is thus 100 in the balloon plot. 
Can you check head(PC.data) and head(CC.data) after applying the gene.scaling function ? 
Or maybe you have an error message or a warning at this step ? 

I hope it helps, let me know!
